.travis.yml

```diff
@@ -14,6 +14,8 @@ script:
   - docker run --rm -it -e TOXENV -e GIT_URL=https://github.com/frenzymadness/python-tox-example.git fedorapython/fedora-python-tox
   # Test in local folder in parallel
   - docker run --rm -it -v $PWD/example_project:/src -w /src -e TOXENV -e TOX_PARAMS="-p auto" fedorapython/fedora-python-tox
+  # Test that we can install (multiple) packages from DNF as devel dependencies; this test overrides the entrypoint, so it doesn't run tox (it always runs system pip)
```

Comment:
It actually runs tox at the end of `run_tests.sh` but it does nothing because there is no configuration for it.

I'd word the comment like this:
```
Test that we can install (multiple) packages from DNF as devel dependencies. Without those, cffi and pyaml are not possible to install from sources.
This test overrides the entrypoint, so it has to be specified manually as the first command. And it always runs system pip.
```

Reply:
Applied couple nitpicks and amedned.